### DISCUSSION TITLE
Side menu expansion state should persist between retries

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#24343] [Q&A]
I don't really remember if this has been reverted at some point...

https://github.com/ppy/osu/assets/75596775/48c24fdb-e9d8-4d74-a0ef-4c383c1f7db1

**Comment by peppy** [comment]
Before we start persisting things at this kind of fine-grained level, a method of doing so with no/zero overhead should be discussed. Because I don't want to have to go around for every interface toggle and add state management logic locally, if possible.
